Resources/Prototypes/Entities/Objects/Tools/fulton.yml

```
@@ -105,3 +105,41 @@
     tags:
       - HideContextMenu
   - type: AnimationPlayer
+
+- type: entity
+  id: ThiefFultonBeacon
+  name: thieving beacon
+  description: A device that sooner or later teleports everything around it to the thief's vault.
```

```suggestion
  description: A device that will teleport everything around it to the thief's vault at the end of the shift.
```
It might be good to be specific here, so people understand how the mechanic works.